Added swedish translation file

erickr opened the pull request:
Added a swedish translation file, could be used for sv_SE and possibly also for sv_FI.

Also added a way to print a list of examples of translations to verify that the translation works well.
Use ```php src/reviewTranslations <locale>``` to print a list of examples using the
locale specified. The list is in no way exclusive, so add more examples to make proofing translations easier.

Feel free to remove/move the reviewTranslations.php if it is not a needed feature.

rlanvin commented:
Thank you very much for your contribution, and apologies it took me so long to review. I decided to merge it to not delay it further, but I think the translations is a bit buggy sometimes. For example:

> hourly at 21h and 22h at minutes 10, 20 and 30, starting from 9/2/97, 9:00 AM, one time

is translated in Swedish:

> varje timme vid 21 och 22.10, 20 och 30, börjar 1997-09-02 09:00, ett tillfälle

Not sure this sentence is correct, but then again, jag kan inte prata svenska flytande.
Feel free to submit another pull request if a fix is necessary.

> Feel free to remove/move the reviewTranslations.php if it is not a needed feature.

This is actually a really cool feature, thanks for coding that. I renamed it, moved it to the "bin" folder with the other utilities script, and added a few things. I'm sure it'll be very helpful for other translators (I know it helped me already reviewing other languages)

erickr commented:
> Thank you very much for your contribution, and apologies it took me so long to review. I decided to merge it to not delay it further, but I think the translations is a bit buggy sometimes. For example:
> 
> > hourly at 21h and 22h at minutes 10, 20 and 30, starting from 9/2/97, 9:00 AM, one time
> 
> is translated in Swedish:
> 
> > varje timme vid 21 och 22.10, 20 och 30, börjar 1997-09-02 09:00, ett tillfälle
Yes, you're right, its not ideal, hardly makes sense. Will see if it can be improved.


> Not sure this sentence is correct, but then again, jag kan inte prata svenska flytande.
> Feel free to submit another pull request if a fix is necessary.
> 
> > Feel free to remove/move the reviewTranslations.php if it is not a needed feature.
> 
> This is actually a really cool feature, thanks for coding that. I renamed it, moved it to the "bin" folder with the other utilities script, and added a few things. I'm sure it'll be very helpful for other translators (I know it helped me already reviewing other languages)

Happy you found use for it, its hard to verify languages in general when you dont speak/write them, but could at least verify no english is coming through to verify the translation is complete. :)
